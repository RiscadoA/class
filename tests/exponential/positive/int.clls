proc main() {
    cut {
        println("!"); !a(b: lint);
        println("let"); let b 42
        |a: ?colint|
        println("?"); ?a;
        println("call 1"); call a(y);
        println(y);
        println("call 2"); call a(z);
        println(z);
        ()
    }
};;
