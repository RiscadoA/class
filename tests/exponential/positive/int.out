!
let
?
call 1
42
call 2
42
